challenges-webapp/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue

```diff
@@ -33,7 +33,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
           <challenges />
         </v-tab-item>
         <v-tab-item v-if="isAdministrator">
-          <realizations id="Realizations" />
+          <realizations id="Realizations" :earner-id="earnerId" />
```

Comment:
Add a boolean attribute to `realizations` component to choose whether to retrieve realizations of all users or not. This will allow to retrieve realizations of current user even when he's an administrator.

```suggestion
          <realizations id="Realizations" :earner-id="earnerId" :retrieve-all="isAdministrator" />
```